qiskit/algorithms/time_evolvers/classical_methods/utils.py

```diff
@@ -0,0 +1,183 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Auxiliary functions for SciPyEvolvers"""
+from typing import List, Tuple, Union
+
+import scipy.sparse as sp
+from scipy.sparse.linalg import expm_multiply
+import numpy as np
+
+from qiskit.quantum_info.states import Statevector
+from qiskit import QuantumCircuit
+from qiskit.opflow import StateFn
```

Comment:
Can we try avoiding opflow here?

Reply:
What would be the correct way of creating something compatible with QuantumCircuit from a numpy array?

Reply:
Would `Statevector` make sense? https://qiskit.org/documentation/stubs/qiskit.quantum_info.Statevector.html#qiskit.quantum_info.Statevector